Formula/rsc_2fa.rb

```diff
@@ -0,0 +1,24 @@
+class Rsc2fa < Formula
+  desc "Two-factor authentication on the command-line"
+  homepage "https://pkg.go.dev/rsc.io/2fa"
+  url "https://github.com/rsc/2fa/archive/refs/tags/v1.2.0.tar.gz"
+  sha256 "d8db6b9a714c9146a4b82fd65b54f9bdda3e58380bce393f45e1ef49e4e9bee5"
+  license "BSD-3-Clause"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args, "-mod=mod", "-o", bin/"2fa"
+  end
+
+  test do
+    out = shell_output("#{bin}/2fa -help 2>&1 || true")
```

Comment:
If this has a non zero return code you can add it as the second argument to shell_output.

```suggestion
    out = shell_output("#{bin}/2fa -help 2>&1")
```